plugins/foreman_remote_execution/index.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: plugin_index
 title: Foreman Remote Execution Documentation
-versions: [0.1, 0.0]
+versions: [nightly, 0.1, 0.0]
```

Comment:
No, but I was just thinking that the contents aren't really of any use to anyone as they're about a non-packaged version.